lib/build.js

```diff
@@ -193,11 +193,21 @@ function build(params) {
                   Effect: 'Allow',
                   Action: [
                     'ec2:*',
-                    'iam:PassRole',
-                    'iam:ListRoles',
-                    'iam:ListInstanceProfiles'
                   ],
                   Resource: '*'
+                },
+                {
+                  Effect: 'Allow',
+                  Action: [
+                    'iam:ListRoles',
+                    'iam:PassRole'
+                  ],
+                  Resource: params.Properties.SpotFleetRequestConfigData.IamFleetRole
+                },
+                {
+                  Effect: 'Allow',
+                  Action: ['iam:ListInstanceProfiles'],
+                  Resource: params.Properties.SpotFleetRequestConfigData.InstanceProfile
```

Comment:
I thought you were gonna have to pull this out of LaunchSpecifications?

Reply:
I realized you can pass anything to magic-cfn-functions in the SpotFleetRequestConfigData, and seemed simpler to pass all the roles the same way.

Reply:
Before you made this change, the parameters that you had to pass to this magic function were _exactly the same_ as what you would pass to a real-life spot fleet resource on cloudformation. After this change, you have introduced one little exception to that rule. Is that a good idea?

Reply:
>After this change, you have introduced one little exception to that rule. Is that a good idea?

When you put it that way, it sure doesn't sound like a good idea. I'll update to point at 
> params.Properties.SpotFleetRequestConfigData.LaunchSpecifications[0].IamInstanceProfile

Which I think should accomplish the same thing and not add a new exception to the params.